appengine/pond/js/blocks.js

```diff
@@ -620,204 +400,16 @@ Blockly.JavaScript['pond_math_single'] = function(block) {
-      Blockly.Procedures.rename), 'NAME')
-      .appendField('(')
-      .appendField('', 'PARAMS')
-      .appendField(') {');
-  this.setStatements_(true);
-  this.appendDummyInput()
-      .appendField('}');
-  this.setMutator(new Blockly.Mutator(['procedures_mutatorarg']));
-  this.setTooltip(Blockly.Msg.PROCEDURES_DEFNORETURN_TOOLTIP);
-  this.arguments_ = [];
-  this.statementConnection_ = null;
-};
-
-/**
- * Define a procedure with a return value.
- * @this Blockly.Block
- */
-Blockly.Blocks['procedures_defreturn'].init = function() {
-  this.setHelpUrl(Blockly.Msg.PROCEDURES_DEFRETURN_HELPURL);
-  this.setColour(Blockly.Blocks.procedures.HUE);
-  var name = Blockly.Procedures.findLegalName(
-      Blockly.Msg.PROCEDURES_DEFRETURN_PROCEDURE, this);
-  this.appendDummyInput()
-      .appendField('function')
-      .appendField(new Blockly.FieldTextInput(name,
-      Blockly.Procedures.rename), 'NAME')
-      .appendField('(')
-      .appendField('', 'PARAMS')
-      .appendField(') {');
-  this.appendValueInput('RETURN')
-      .setAlign(Blockly.ALIGN_RIGHT)
-      .appendField('return');
-  this.appendDummyInput()
-      .appendField('}');
-  this.setMutator(new Blockly.Mutator(['procedures_mutatorarg']));
-  this.setTooltip(Blockly.Msg.PROCEDURES_DEFRETURN_TOOLTIP);
-  this.arguments_ = [];
-  this.setStatements_(true);
-  this.statementConnection_ = null;
-};
-
-Blockly.Msg.PROCEDURES_BEFORE_PARAMS = '';
-
-/**
- * Call a procedure with no return value.
- * @this Blockly.Block
- */
-Blockly.Blocks['procedures_callnoreturn'].init = function() {
-  this.setHelpUrl(Blockly.Msg.PROCEDURES_CALLNORETURN_HELPURL);
-  this.setColour(Blockly.Blocks.procedures.HUE);
-  this.appendDummyInput()
-      .appendField('', 'NAME')
-      .appendField('(');
-  this.appendDummyInput('TAIL')
-      .appendField(');');
-  this.setInputsInline(true);
-  this.setPreviousStatement(true);
-  this.setNextStatement(true);
-  this.setTooltip(Blockly.Msg.PROCEDURES_CALLNORETURN_TOOLTIP);
-  this.arguments_ = [];
-  this.quarkConnections_ = {};
-  this.quarkArguments_ = null;
-};
-
-/**
- * Modify this block to have the correct number of arguments.
- * @private
- * @this Blockly.Block
- */
-Blockly.Blocks['procedures_callnoreturn'].updateShape_ = function() {
-  for (var i = 0; i < this.arguments_.length; i++) {
-    if (!this.getInput('ARG' + i)) {
-      // Add new input.
-      var field = new Blockly.FieldLabel(this.arguments_[i]);
-      var input = this.appendValueInput('ARG' + i);
-      if (i > 0) {
-        input.appendField(',');
-      }
-      input.init();
-    }
-  }
-  // Remove deleted inputs.
-  while (this.getInput('ARG' + i)) {
-    this.removeInput('ARG' + i);
-    i++;
-  }
-  this.moveInputBefore('TAIL', null);
-};
+Blockly.JavaScript['pond_math_number'] = Blockly.JavaScript['math_number'];
 
 /**
- * Call a procedure with a return value.
- * @this Blockly.Block
+ * Defining aliases to support blockly xml generated with old naming.
```

Comment:
Uppercase XML.  Also add a date, so that our grandchildren will know that this refers to a deprecation that occurred 50 years ago, not a week ago.  See the examples in this file:
https://github.com/google/blockly/blob/master/appengine/redirect.html#L7